docs/pages/application-access/getting-started.mdx

```diff
@@ -67,45 +84,60 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-<Tabs>
-<TabItem label="Docker">
+## Step 2/5. Start Grafana
 
-## Step 2/4. Start Grafana
+You can start Grafana in a Docker container or Kubernetes cluster by running the appropriate steps.
+For more information, click **Docker** or **Kubernetes cluster**.
```

Comment:
```suggestion
```

IMO we can trust the reader to know to click on the correct tab for their environment of choice.